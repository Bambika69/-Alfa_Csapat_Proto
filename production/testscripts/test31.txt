createfield
createvirologist F1
createequipment bag -v V1
startturn V1
toggle B1