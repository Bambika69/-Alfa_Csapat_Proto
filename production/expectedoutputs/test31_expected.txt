Field created:
ID: F1
Type: Field
Neighbours: null

Virologist created:
ID: V1
Field: F1
Backpack: B1

Equipment created:
ID: E1
Type: bag
Destination: V1
Result: Successful

V1's turn started.

Equipment toggled on Virologist:
Equipment: E1
Virologist: V1
Inf: unwear
